instrumentation/net/http/otelhttp/internal/request/resp_writer_wrapper.go

```diff
@@ -61,7 +61,7 @@ func (w *RespWriterWrapper) Write(p []byte) (int, error) {
 
 // WriteHeader persists initial statusCode for span attribution.
 // All calls to WriteHeader will be propagated to the underlying ResponseWriter
-// and will persist the statusCode from the first call.
+// and will persist the statusCode from the first call (except for informational response status codes).
```

Comment:
This code is auto-generated (see the top of the file).
You need to modify the template and run `make generate` locally.
https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/internal/shared/request

Reply:
Good catch, don't know how I've missed that. Fixed and regenerated with `make generate`.